src/components/InvoicesList/InvoicesList.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import { Loading } from '../Loading/Loading';
+import { InjectAppServices } from '../../services/pure-di';
+import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+
+const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
+  const [stateInvoices, setStateInvoices] = useState({ loading: true });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setStateInvoices({ loading: true });
+      const invoicesResponse = await dopplerBillingApiClient.getInvoices(0, 0);
+      if (!invoicesResponse.success) {
+        setStateInvoices({ loading: false, items: [] });
+      } else {
+        setStateInvoices({
+          loading: false,
+          items: invoicesResponse.value.items,
+          totalItems: invoicesResponse.value.totalItems,
+        });
+      }
+    };
+
+    fetchData();
+  }, [dopplerBillingApiClient]);
+
+  return (
+    <>
+      <Helmet>
+        <meta name="invoices" />
+      </Helmet>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <h2>Facturas emitidas</h2>
```

Comment:
only use 1 language because its for us? or why?

Reply:
We will suport different language in the future PRs. It was initial PR with minimal functionallity. Thanks for your feedback